Remotion/Data/DomainObjects/Mapping/ClassDefinition.cs

```diff
@@ -446,61 +216,63 @@ private void FillAllDerivedClasses (List<ClassDefinition> allDerivedClasses)
       }
     }
 
-    private void CheckPropertyDefinitions (IEnumerable<PropertyDefinition> propertyDefinitions)
+    public override string ToString ()
+    {
+      return $"{GetType().GetFullNameChecked()}: {ID}";
+    }
```

Comment:
Since you're doing cleanup: switch to expression bodied member?
also noticed: someone thought that the fullname instead of the name would be a good idea here. I don't think that's true anymore in 2022. You want to keep it or change it?